cartridges/int_cloudinary_sfra/cartridge/controllers/Product.js

```diff
@@ -107,8 +107,8 @@ server.append('Variation', function (req, res, next) {
             colorAttrValueID = params.variables.color.value;
         }
 
-        if (params && params.variables && params.variables.accessorySize && params.variables.accessorySize.value || params.variables.size && params.variables.size.value) {
-            sizeAttrValueID = params.variables.accessorySize.value ? params.variables.accessorySize.value : params.variables.size.value ? params.variables.size.value : null
+        if ((params && params.variables) && ((params.variables.accessorySize && params.variables.accessorySize.value) || (params.variables.size && params.variables.size.value))) {
+            sizeAttrValueID = params.variables.accessorySize.value || params.variables.size.value || null;
```

Comment:
@asad-rafter no need for "|| null" as this will never happen with the above condition